www_src/pages/page/page.jsx

```diff
@@ -164,6 +166,14 @@ var Page = React.createClass({
           state.currentElementId = id;
         }
         this.setState(state);
+
+        //Ensure we don't reach a race conditions where the buttons are
+        //re-enabled before the menu is hidden. 200ms matches the css animation
+        //speed.
+
+        setTimeout(function() {
```

Comment:
Can you please provide some guidance / links / resources for @errietta to take a look at?